### PR TITLE
Update synology_srm component to use the new config flow workflow

### PR DESCRIPTION
**Pull request by aerialls**
## Proposed change

Update the documentation to reflect changes for the Synology SRM component.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35412
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/synology_srm.markdown [hunk 1]
```diff
@@ -13,52 +13,15 @@ This platform allows you to detect presence by looking at connected devices to a
 
 ## Configuration
 
-To use this device tracker in your installation, add the following to your `configuration.yaml` file:
+<div class='note'>
+The Synology SRM integration can only be configured through the user interface since Home Assistant 0.110. The legacy YAML configuration is not supported anymore.
+</div>
```
Comment (frenck) [inline source/_integrations/synology_srm.markdown]: We don't keep a changelog in our documentation. Please remove this note.

File: source/_integrations/synology_srm.markdown [hunk 2]
```diff
@@ -13,52 +13,15 @@ This platform allows you to detect presence by looking at connected devices to a
...
-  required: false
-  default: true
-  type: boolean
-verify_ssl:
-  description: Enable or disable SSL certificate verification.
-  required: false
-  default: false
-  type: boolean
-{% endconfiguration %}
+The integration can be configured through the user interface at *Configuration* -> *Integrations*. Search for "Synology SRM", fill in the configuration form with your username and password, and then click **Submit**.
```
Comment (frenck) [inline source/_integrations/synology_srm.markdown]: I believe there is a step missing, where the user needs to click something to be able to search for "Synology SRM"...

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @aerialls! 👍 

I've reviewed it and left some comments. Could you please take a look?

Thanks 👍

**Comment by aerialls** [comment]
Hello @frenck, thanks for the feedback! I've updated my PR with your remarks.

**Comment by aerialls** [comment]
Hello @frenck. Did you have time to review my changes and also the main PR on `core`? Thanks!

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**